File: src/ChpokkWeb/SystemFiles/Templates/ProjectTemplates/CSharp/Silverlight/1041/BusinessApplication/BA.Web/Models/Shared/User.shared.cs
namespace $safeprojectname$
{
    /// <summary>
    /// 共有プロパティおよびメソッドを追加する User 型を拡張する部分クラスです。
    /// これらのプロパティおよびメソッドは、サーバー アプリケーションとクライアント アプリケーションの両方で使用できます。
    /// </summary>
    public partial class User
    {
        /// <summary>
        /// ユーザーの表示名 (既定ではユーザーの FriendlyName) を返します。
        /// FriendlyName が設定されていない場合は、ユーザー名が返されます。
        /// </summary>
        public string DisplayName
        {
            get
            {
                if (!string.IsNullOrEmpty(this.FriendlyName))
                {
                    return this.FriendlyName;
                }
                else
                {
                    return this.Name;
                }
            }
        }
    }
}

File: src/ChpokkWeb/SystemFiles/Templates/ProjectTemplates/VisualBasic/Silverlight/1033/BusinessApplication/BA.Web/Models/Shared/User.shared.vb
Namespace Web

    Partial Public Class User
        ''' <summary>
        ''' Returns the user display name, which by default is its FriendlyName.
        ''' If FriendlyName is not set, the User Name is returned.
        ''' </summary>
        Public ReadOnly Property DisplayName() As String
            Get
                If Not String.IsNullOrEmpty(Me.FriendlyName) Then
                    Return Me.FriendlyName
                Else
                    Return Me.Name
                End If
            End Get
        End Property
    End Class
End Namespace